hack/shred_accounts.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+ACCOUNT_OPERATOR_NAMESPACE=aws-account-operator
+ACCOUNT_SHREDDER_NAMESPACE=aws-account-shredder
+OC_LOG_WINDOW="1h"
+
+
+function help {
+  name=`basename $0`
```

Comment:
Can you change this to `$(basename $0)` to keep it consistent with the rest of the file. I also find backticks pretty hard to read and parse in scripts so i tend to never use them anywhere.